tdrs-backend/tdpservice/search_indexes/parsers/tanf_validators.py

```diff
@@ -0,0 +1,191 @@
+"""Validators for TANF data parser."""
+
+# T1 Category 2 TANF Fatal Edits
+# https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_fatal_edits_sections_1_and_4.pdf
+
+def t1_003(model_obj):
+    """Validate stratum."""
+    return _get_field_by_item_number(model_obj, '5') > 0 and _get_field_by_item_number(model_obj, '5') < 100
+
+def t1_006(model_obj):
+    """Validate report month."""
+    month = model_obj.RPT_MONTH_YEAR
+    return int(str(month)[0:4]) >= 1998
```

Comment:
I only used the _get_field_by_item_number a couple functions to show what it would look like. It will be easier to make the validator functions with _get_field_by_item_number, but they might be harder to read. 

However, the field value could be put in the comment.